people/zhoujie.py

```diff
@@ -0,0 +1,6 @@
+import numpy as np 
+ 
+#Converts the list of tuples to ndarray
+list_tuples = [(1,2,3),(4,5)] 

```

Comment:
please check if this line of code meets the PEP-8 code style, as I mentioned in the tutorial. If not, please modify it by pushing a new commit to the same branch (it will show here automatically on the web); if already meet, please click "resolve".

Reply:
Sorry@dragonyanglong, After learning the standard of PEP-8, I changed the style of indentation, and added prefixes to the description when committed. Please review and merge again.